### PR TITLE
Allow customizing artifact name for build-uki command

### PR DESCRIPTION
**Pull request by ZauberNerd**
The `build-iso` command accepts a `--name` flag to specify the basename
of the generated iso file, while the `build-uki` command uses the value
of `KAIROS_RELEASE` from the `/etc/os-release` file.
This commit introduces a `--name` flag for the `build-uki` command to
allow specifying the basename of the generated .iso or .tar file.
When the output type is `uki` the `--name` flag is ignored, because the
uki output type write the uki filesystem into the output folder.

### CODE REVIEW

**Review by Itxaka — APPROVED** [review]

File: pkg/action/build-uki.go [hunk 1]
```diff
@@ -148,7 +150,7 @@ func (b *BuildUKIAction) Run() error {
 		}
 		// Then build the image
 
-		err = b.createContainer(b.outputDir, kairosVersion)
+		err = b.createContainer(b.outputDir, kairosVersion, b.name)
```
Comment (jimmykarily) [inline pkg/action/build-uki.go:153]: `b` is available in `createContainer` the same way it happens in `createISO` above. Why pass it as an argument here?

Comment (ZauberNerd) [reply pkg/action/build-uki.go:153]: Ah, good catch. I oriented myself on the other parameters (`b.outputDir, kairosVersion`).
I'll update the PR.

Comment (ZauberNerd) [reply pkg/action/build-uki.go]: Updated

**Comment by Itxaka** [comment]
thanks @ZauberNerd !

**Pull request merged**